README.md

```diff
@@ -13,6 +13,22 @@
 - [GoogleTest](https://github.com/google/googletest) (optional; bundled) - unit testing
 - Doxygen (optional) - documentation
 
+## How to install DLA-Future with spack
+
+We provide a spack package DLA-Future that can be easily added to your own spack as follows:
+
+`spack repo add <dlaf root>/spack`
```

Comment:
I would standardize the `<dlaf root>`: here we are using it like this, while later in the document we use `$DLAF_ROOT`.

Reply:
Alright, lets stick to `$DLAF_ROOT` then.